.github/pull_request_template.md

```diff
@@ -1,12 +1,14 @@
 <!--
 
-Dear GLPI user.
+Dear GLPI developer.
 
-BEFORE SUBMITTING YOUR ISSUE, please make sure to read and follow these steps:
+BEFORE SUBMITTING YOUR PULL REQUEST, please make sure to read and follow these steps:
 
 * We don't support community plugins. Contact directly their authors, or use the community forum : http://forum.glpi-project.org.
 * For feature requests or enhancements, use the suggest dedicated site (http://suggest.glpi-project.org). We check it very often.
 * We prefer to keep this tracker in ENGLISH. If you want support in your language, the community forum (http://forum.glpi-project.org) is the best place.
+* Evolutions and features should target the `main` branch and should be discussed in a issue before submitting a PR.
```

Comment:
I really like this part you have added. While we are glad for external contributions, it is true that it is not so easy to review them if they were done without any prior discussions (especially for big features).

Maybe it would be even better to make it part of the checklist ? I fear the comment might be too easily ignored.
Could be something like `[ ] As an external contribuer, I have submitted an issue to discuss this change here: #XXXX`.

What do you think ?

Reply:
Also, maybe we need a new dedicated issue template for this case ?

Reply:
Indeed about the missing template, I will try to do that later in the day.

I'm not sure about insisting on the process for external contributors, if they don't want to read, making twice the statement will not help.
I prefer having the template as short as possible.